rcl/test/rcl/test_timer.cpp

```diff
@@ -459,7 +459,8 @@ TEST_F(TestTimerFixture, test_ros_time_wakes_wait) {
 
   std::thread wait_thr([&](void) {
       rcl_wait_set_t wait_set = rcl_get_zero_initialized_wait_set();
-      ret = rcl_wait_set_init(&wait_set, 0, 0, 1, 0, 0, context_ptr, rcl_get_default_allocator());
+      ret =
+      rcl_wait_set_init(&wait_set, 0, 0, 1, 0, 0, 0, context_ptr, rcl_get_default_allocator());
```

Comment:
nitpick: you have to indent here, if that makes it too long, then you need to break the line in another way

Reply:
If I remember correctly, I introduced the line break with an indent, but then uncrustify removed the indent.

I will break the line in a different way.